pkg/client.go

```diff
@@ -41,9 +41,23 @@ const (
 	SCOPE_PING_READ               = "/ping.read"
 )
 
+const (
+	baseURL            = "https://api.moov.io"
+	pathBankAccounts   = "accounts/%s/bank-accounts"
+	pathCards          = "accounts/%s/cards"
+	pathApplePay       = "accounts/%s/apple-pay"
+	pathPaymentMethods = "accounts/%s/payment-methods"
+	pathWallets        = "accounts/%s/wallets"
+	pathWalletTrans    = "accounts/%s/wallets/%s/transactions"
+	pathTransactions   = "accounts/%s/transactions"
+	pathTransfers      = "transfers"
+	pathDisputes       = "disputes"
```

Comment:
Why do we need these constants? Can't we just write the URLs directly where they're used?